cmd/datool/datool.go

```diff
@@ -92,6 +92,7 @@ type ClientStoreConfig struct {
 	SigningWallet         string        `koanf:"signing-wallet"`
 	SigningWalletPassword string        `koanf:"signing-wallet-password"`
 	MaxStoreChunkBodySize int           `koanf:"max-store-chunk-body-size"`
+	EnableChunkedStore    bool          `koanf:"enable-chunked-store"`
```

Comment:
It is a little confusing having to do `enable-chunked-store=false`, how about changing the option to `disable-chunked-store`?

Reply:
This is a recurring theme, and, in general, I prefer flags/bools to be worded in a positive sense. I think @eljobe was the first one to bring this up.